src/radiation/mocmc.cpp

```diff
@@ -128,8 +128,7 @@ void MOCMCInitSamples(T *rc) {
       },
       Kokkos::Sum<int>(nsamp_tot));
 
-  ParArrayND<int> new_indices;
-  auto new_mask = swarm->AddEmptyParticles(nsamp_tot, new_indices);
+  auto new_mask = swarm->AddEmptyParticles(nsamp_tot);
```

Comment:
```suggestion
  auto new_particles_context = swarm->AddEmptyParticles(nsamp_tot);
```
Just because the new context struct is technically more than a mask now.